packages/vite/src/types/terser.d.ts

```diff
@@ -245,6 +249,16 @@ export namespace Terser {
     includeSources?: boolean
     filename?: string
     root?: string
+    asObject?: boolean
     url?: string | 'inline'
   }
+
+  export function minify(
+    files: string | string[] | { [file: string]: string },
+    options?: MinifyOptions,
+  ): Promise<MinifyOutput>
+  export function minify_sync(
+    files: string | string[] | { [file: string]: string },
+    options?: MinifyOptions,
+  ): MinifyOutput
```

Comment:
I don't think we need this part as we don't rely on it. It should be safe to remove this.

Reply:
I wasn't sure if they were new functionality. I've removed them :)